src/main/java/io/github/jamsesso/jsonlogic/JsonLogic.java

```diff
@@ -100,6 +100,10 @@ public static boolean truthy(Object value) {
     }
 
     if (value instanceof Number) {
+      if (value instanceof BigDecimal) {
+        return ((BigDecimal) value).compareTo(BigDecimal.ZERO) != 0;
```

Comment:
How is truthy evaluated ? Are negative numbers considered as truthy ?

Reply:
All non-zero numbers are considered as true, and zero is considered as false
Following the same logic as theirs
https://github.com/TogaiHQ/json-logic-java/blob/7c93e1584c3cfdd6e15dbf40479d6a9ca94534ef/src/main/java/io/github/jamsesso/jsonlogic/JsonLogic.java#L129